src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
```

Comment:
The `<form>` tag is self-closing, which is not valid HTML syntax. Please change it to have an opening `<form>` and a closing `</form>` tag.